pulsar-broker/src/test/java/org/apache/pulsar/broker/TimeAverageBrokerDataTest.java

```diff
@@ -0,0 +1,138 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pulsar.broker;
+
+import org.apache.pulsar.policies.data.loadbalancer.NamespaceBundleStats;
+import org.junit.Test;
+import org.testng.Assert;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+
+public class TimeAverageBrokerDataTest {
+
+    @Test
+    public void testIllegalArg() {
+        TimeAverageBrokerData timeAverageBrokerData = new TimeAverageBrokerData();
+        Assert.assertThrows(NullPointerException.class, () -> timeAverageBrokerData.reset(null, null, null));
+    }
+
+    @Test
+    public void testResetMethodOfOneBundlesEmptyBundleMap() {
+        TimeAverageBrokerData timeAverageBrokerData = new TimeAverageBrokerData();
+        Set<String> bundles = new HashSet<>();
+        bundles.add("a");
+        Map<String, BundleData> emptyMap = new HashMap<>();
+        NamespaceBundleStats namespaceBundleStats = new NamespaceBundleStats();
+        namespaceBundleStats.msgThroughputIn = 1;
+        namespaceBundleStats.msgThroughputOut = 2;
+        namespaceBundleStats.msgRateIn = 3;
+        namespaceBundleStats.msgRateOut = 4;
+        timeAverageBrokerData.reset(bundles, emptyMap, namespaceBundleStats);
+        Assert.assertEquals(timeAverageBrokerData.getShortTermMsgThroughputIn(), 1);
```

Comment:
using static import for Assert.assertEquals would improve readability of the test.

Reply:
Fixed :)